vars/virtualenv.groovy

```diff
@@ -1,6 +1,11 @@
 import com.ableton.VirtualEnv
 
 
+/**
+ * Create a new {@link com.ableton.VirtualEnv} instance.
+ * @param python Python version to use.
```

Comment:
We should maybe rename that variable to `pythonVersion`?

Reply:
No, since this value can also be a path. I've fixed the documentation accordingly.